website/cue/reference/releases/0.47.0.cue

```diff
@@ -0,0 +1,427 @@
+			description: """
+				Fix a Vector crash that occurred when the internal metrics generated a lot of groups by increasing groups max limit from 128 to 256.
+				"""
+			contributors: ["triggerhappy17"]
+		},
+		{
+			type: "feat"
+			description: """
+				Allows users to specify AWS authentication and the AWS service name for HTTP sinks to support AWS API endpoints that require SigV4.
+				"""
+			contributors: ["johannesfloriangeiger"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add support for fluentd forwarding over a unix socket
+				"""
+			contributors: ["tustvold"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Replaced miniz_oxide backend for zlib compression with zlib-rs for significantly improved compression performance. zlib-rs provides 2-3x performance improvements for compression, and ~20% improvements for decompression than miniz_oxide.
+				"""
+			contributors: ["JakubOnderka"]
+		},
+		{
+			type: "feat"
+			description: """
+				Add ACK support to message buffering feature of `websocket_server` sink, allowing this component to cache latest received messages per client.
+				"""
+			contributors: ["esensar", "Quad9DNS"]
+		},
+		{
+			type: "chore"
+			description: """
+				Added a new `extra_headers` option to `greptimedb_logs` sink configuration to set additional headers for outgoing requests.
+				
+				change `greptimedb_logs` sink default content type to `application/x-ndjson` to match the default content type of `greptimedb` sink
+				if you use the greptimedb version v0.12 or earlier, you need to set the content type to `application/json` in the sink configuration
+				
+				Example:
+				
+				```yaml
+				sinks:
+				  greptime_logs:
+				    type: greptimedb_logs
+				    inputs: ["my_source_id"]
+				    endpoint: "http://localhost:4000"
+				    table: "demo_logs"
+				    dbname: "public"
+				    extra_headers:
+				      x-source: vector
+				```
+				
+				```toml
+				[sinks.greptime_logs]
+				type = "greptimedb_logs"
+				inputs = ["my_source_id"]
+				endpoint = "http://localhost:4000"
+				table = "demo_logs"
+				dbname = "public"
+				
+				[sinks.greptime_logs.extra_headers]
+				x-source = "vector"
+				```
+				"""
+			contributors: ["greptimedb"]
+		},
+		{
+			type: "feat"
+			description: """
+				This change introduces a new configuration option in the StatsD source: `convert_to` of type `ConversionUnit`. By default, timing values in milliseconds (`ms`) are converted to seconds (`s`). Users can set `convert_to` to `"milliseconds"` to preserve the original millisecond values.
+				"""
+			contributors: ["devkoriel"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed `file` source bug where known small files were not deleted after the specified `remove_after_secs`.
+				"""
+			contributors: ["linw1995"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed a `AwsAuthentication::File` bug where `region` was missing from the `STS` authentication endpoint.
```

Comment:
Suggestion: write this as a user-facing change. They aren't going to know what `AwsAuthentication::File` is.